tethysapp/gwdm/public/js/main.js

```diff
@@ -38,7 +38,7 @@ function addErrorMessage(error, div_id) {
         div_id_string = '#'+div_id;
     }
     $(div_id_string).html(
-        '<span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>' +
+        '<span class="exclamation-circle-fill" aria-hidden="true"></span>' +
```

Comment:
Need to change `<span>` to `<i>` and add `bi` to classes:

e.g.:
```
'<i class="bi bi-exclamation-circle-fill" aria-hidden="true"></i>' +
```

Fix all instances of this in this file.